### PR TITLE
Notice - Inform that the project was deprecated and no longer maintained

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**
This project is deprecated and is no longer supported or maintained for a while. 
We must make that clear and let the users know how they can do tests currently.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,65 @@
+# Notice 
+---
+**This project is deprecated and is no longer supported or maintained.**
+
+All tests and checks made by this project were moved to the project [operator-framwork/api](https://github.com/operator-framework/api) and specifically to the validator [OperatorHub](https://github.com/operator-framework/api/blob/v0.10.7/pkg/validation/internal/operatorhub.go). Note that any new requirement has been addresssed to the [operator-framwork/api](https://github.com/operator-framework/api).
```
Comment (kevinrizza) [inline README.md:5]: This really only calls out the validation portion of courier, which wasn't really the core use of the project -- it was used as a way to package and push manifests to an AppRegistry endpoint. If we're going to give users specific steps for validation, we should probably at least mention that the AppRegistry packaging format is no longer supported in the project and point them to the bundle format and the Operator Registry at the same time.

**Review by kevinrizza — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (camilamacedo86) [reply README.md]: @kevinrizza thank you for the review. Please, feel free to re-check this one.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
+# Notice 
+
+:warning: **This project is deprecated and is no longer supported or maintained.**
+
+AppRegistry packaging format is no longer supported by [OLM](https://github.com/operator-framework/operator-lifecycle-manager).
+You can use the [Bundle](https://github.com/operator-framework/operator-registry/blob/v1.19.5/docs/design/operator-bundle.md) format and [Operator Registry](https://github.com/operator-framework/operator-registry) instead.
+
+**What about `operator-courier verify`? How can I verify the manifests?**  
+
+All tests and checks made by this project were moved to the project [operator-framwork/api](https://github.com/operator-framework/api) and specifically to the validator [OperatorHub](https://github.com/operator-framework/api/blob/v0.10.7/pkg/validation/internal/operatorhub.go). You can use the [operator-framwork/api](https://github.com/operator-framework/api) directly or use the [Operator-SDK](https://github.com/operator-framework/operator-sdk) CLI to do these checks with the command `operator-sdk bundle validate ./bundle --select-optional name=operatorhub`([More info](https://sdk.operatorframework.io/docs/cli/operator-sdk_bundle_validate/)).
```
Comment (camilamacedo86) [inline README.md:10]:
```suggestion
All tests and checks made by this project were moved to the project [operator-framwork/api](https://github.com/operator-framework/api) and specifically to the validator [OperatorHub](https://github.com/operator-framework/api/blob/v0.10.7/pkg/validation/internal/operatorhub.go). You can use the [operator-framwork/api](https://github.com/operator-framework/api) directly or [Operator-SDK](https://github.com/operator-framework/operator-sdk) to do these checks with the command `operator-sdk bundle validate ./bundle --select-optional name=operatorhub`([More info](https://sdk.operatorframework.io/docs/cli/operator-sdk_bundle_validate/)).
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+
+:warning: **This project is deprecated and is no longer supported or maintained.**
+
+AppRegistry packaging format is no longer supported by [OLM](https://github.com/operator-framework/operator-lifecycle-manager).
+You can use the [Bundle](https://github.com/operator-framework/operator-registry/blob/v1.19.5/docs/design/operator-bundle.md) format and [Operator Registry](https://github.com/operator-framework/operator-registry) instead.
+
+**What about `operator-courier verify`? How can I verify the manifests?**  
+
+All tests and checks made by this project were moved to the project [operator-framwork/api](https://github.com/operator-framework/api) and specifically to the validator [OperatorHub](https://github.com/operator-framework/api/blob/v0.10.7/pkg/validation/internal/operatorhub.go). You can use the [operator-framwork/api](https://github.com/operator-framework/api) directly or [Operator-SDK](https://github.com/operator-framework/operator-sdk) to do these checks with the command `operator-sdk bundle validate ./bundle --select-optional name=operatorhub`([More info](https://sdk.operatorframework.io/docs/cli/operator-sdk_bundle_validate/)).
+
```
Comment (camilamacedo86) [inline README.md:11]: @kevinrizza could we get this one merged now? WDYT

**Pull request merged**